azext_edge/edge/providers/orchestration/resources/connector/opcua/certs.py

```diff
@@ -561,7 +557,13 @@ def _check_secret_name(
 
```

Comment:
This can be just `return`.

Also because the function can return None `_check_secret_name` return type hint should be `Optional[str]`